model/src/main/kotlin/config/PostgresConnection.kt

```diff
@@ -53,18 +54,21 @@ data class PostgresConnection(
      * The full path of the certificate file.
      * See: https://jdbc.postgresql.org/documentation/head/connect.html
      */
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
Shouldn't instead the whole class be annotated with `@JsonInclude(JsonInclude.Include.NON_NULL)` or even `@JsonInclude(JsonInclude.Include.NON_DEFAULT)`? (The latter probably hides too much what a user might be interested to see in the active configuration.)

Reply:
I decided for per-property because I thought it is more obvious / harder to miss that irrelevant properties shall be excluded. 
Does this make sense to you as well?

Reply:
Per-property is fine with me, but I still believe it should be `NON_NULL` instead of `NON_EMPTY`, or?